Neos.Flow/Classes/Http/Helper/UriHelper.php

```diff
@@ -11,15 +11,21 @@
  * source code.
  */
 
+use Neos\Utility\Arrays;
 use Psr\Http\Message\UriInterface;
 
 /**
  * Helper to extract information from Uris.
  */
-abstract class UriHelper
+final class UriHelper
```

Comment:
I'd personally prefer the other helper, this Helper namespace seems rather pointless.

Reply:
Yeah but there are already a few helpers inside: https://github.com/neos/flow-development-collection/tree/9.0/Neos.Flow/Classes/Http/Helper

and it would be a little less breaking (though I think no one uses the helper)

The other helper (from the merge yesterday) resided directly in the http namespace but at that point I was unaware of the other helpers

Reply:
I don’t exactly love the namespace either but idk it does no harm as well.

Btw _you_ added the uri helper as part of the psr7 migration in this place ;) so better keep that location?

Reply:
Right, I guess better keep the older one, I also just realized that. Shame on me for creating that useless namespace, although back then I planned for more helpers to support using PSR-7 in Flow (but pretty much everything else exists in guzzle so no need for them). tl;dr; kk lets keep this.